docs/modules/ROOT/pages/vshn-managed/postgresql/create.adoc

```diff
@@ -38,6 +38,19 @@ spec:
 
 NOTE: To get more information about all available configuration options, please see the xref:references/crds.adoc#k8s-api-github-com-vshn-component-appcat-apis-vshn-v1-vshnpostgresql[API Reference]
 
+=== Sidecar Overhead and Fair-Use Policy
+
+Each PostgreSQL Standalone instance has following overhead:
+
+* CPU: 364m
+* Memory: 704Mi
+
+These are for various sidecar containers that run alongside the actual PostgreSQL instance.
+These containers handle connection pooling, metrics and other internal functionality.
+
+NOTE: Please be aware that the normal https://docs.appuio.cloud/user/explanation/fair-use-policy.html[Fair Use Policy] applies to AppCat Services by VSHN as well.
+When chosing memory and CPU for your instance please keep the overhead in mind.
```

Comment:
What do you mean overhead in this context?

Reply:
The overhead of the sidecars as described a few lines above that one.

Reply:
ok, it's important that it's clear :)